### PR TITLE
add multi-tenancy compatibility

### PR DESCRIPTION
**Pull request by Twister42**
A small modification so we can use multi-tenancy with the lib

### CODE REVIEW

File: src/firebaseAdmin.js [hunk 1]
```diff
@@ -16,6 +16,17 @@ const getFirebasePublicAPIKey = () => {
   return config.firebaseClientInitConfig.apiKey
 }
 
+/**
+ * Get the firebase admin TenantAwareAuth or the BasicAuth object needed for the user
+ */
+const getAuth = (AuthUser) => {
+  const admin = getFirebaseAdminApp()
+  if (AuthUser.tenantId) {
+    return admin.auth().tenantManager().authForTenant(AuthUser.tenantId)
```
Comment (kmjennison) [inline src/firebaseAdmin.js:25]: Please add test coverage for this.

File: src/initFirebaseClientSDK.js [hunk 2]
```diff
@@ -11,6 +11,9 @@ export default function initFirebaseClientSDK() {
       )
     }
     firebase.initializeApp(firebaseClientInitConfig)
+    if (firebaseClientInitConfig.tenantId) {
+      firebase.auth().tenantId = firebaseClientInitConfig.tenantId
```
Comment (kmjennison) [inline src/initFirebaseClientSDK.js:15]: Please add test coverage for this.

**Review by kmjennison — CHANGES_REQUESTED** [review]
Thanks! Please add tests to cover the tenant logic (see code coverage comments) or let me know if you run into any trouble.

**Comment by locona** [comment]
@kmjennison 
Why did you close this one?

**Comment by Twister42** [comment]
Probably because I never made the unit tests...
I didn't find the time yet. I need to release my app first :)

**Comment by locona** [comment]
@Twister42 

I understand!

May I create a test and submit my PR when you have time?

**Comment by Twister42** [comment]
Sure ! I didn't commit the tests because I didn't find a good way to do them (I'm quite noob in JS and TS)
But if you have time and is willing to make them you're most welcome I'll add them to the PR as fast as I can

**Comment by locona** [comment]
OK! Thanks !

**Comment by kmjennison** [comment]
Yes, I closed this because it had been inactive for a while and has merge conflicts, but I welcome an updated PR. Thank you both for contributing!

**Pull request closed without merging**